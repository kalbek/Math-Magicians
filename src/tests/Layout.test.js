import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Layout from '../components/Layout';

test('It should render correctly', () => {
  const tree = render(
    <BrowserRouter>
      <Layout />
    </BrowserRouter>,
  );
  expect(tree).toMatchSnapshot();
});
