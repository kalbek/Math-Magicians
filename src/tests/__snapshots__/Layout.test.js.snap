// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="container flex-spaced"
        >
          <div
            class="title"
          >
            <h1>
              Math Magicians
            </h1>
          </div>
          <div
            class="menu flex gap-1"
          >
            <a
              class="link"
              href="/"
              name="home"
            >
              <div
                class="Home ptr"
              >
                Home
              </div>
            </a>
            |
            <a
              class="link"
              href="/Calculator"
              name="calculator"
            >
              <div
                class="calc ptr"
              >
                Calculator
              </div>
            </a>
            |
            <a
              class="link"
              href="/Quotes"
              name="quotes"
            >
              <div
                class="Quote ptr"
              >
                Quote
              </div>
            </a>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="container flex-spaced"
      >
        <div
          class="title"
        >
          <h1>
            Math Magicians
          </h1>
        </div>
        <div
          class="menu flex gap-1"
        >
          <a
            class="link"
            href="/"
            name="home"
          >
            <div
              class="Home ptr"
            >
              Home
            </div>
          </a>
          |
          <a
            class="link"
            href="/Calculator"
            name="calculator"
          >
            <div
              class="calc ptr"
            >
              Calculator
            </div>
          </a>
          |
          <a
            class="link"
            href="/Quotes"
            name="quotes"
          >
            <div
              class="Quote ptr"
            >
              Quote
            </div>
          </a>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
